### ISSUE TITLE
"send to client" should succeed when TR_AUTH is not set

### ISSUE THREAD

**Issue by mklbtz** [kryptxy/torrench#73]
The documentation says that if `TR_AUTH` is "not set" that it will use the default of no username or password. When I was first setting up torrench, I assumed that meant that I didn't need the env variable at all, but I kept getting this message when I tried sending to `transmission-remote`.

    The TR_AUTH environment variable is not set

I was able to resolve the issue by running `export TR_AUTH=""`. That's fine, it works, but it would improve the install experience if torrench could just interpret a _missing_ environment variable the same as an _empty_ one. At the very least, clarifying the docs would help.

**Comment by adrianmihalko** [comment]
Same problem here.

**Comment by datafanatic** [comment]
Hi, I'll see if I can tackle this during the weekend but I know if I'll be able to get it merged.

**Comment by mklbtz** [comment]
@datafanatic let me know if I can help somehow.

**Comment by datafanatic** [comment]
@mklbtz Sure enough. I haven't been able to touch this since I'm busy with another programming project, but I haven't forgotten about it. Just a heads up.

**Comment by kryptxy** [comment]
Thanks @mklbtz for reporting. Setting up `$TR_AUTH` would be necessary.
It's a little more complicated than I thought. At the time of writing (the docs), I had tested this on Arch Linux. In Arch, at that time I was able to access transmission-daemon without any username/password (Thus mentioning Username/Password as None in docs). What I didn't do was test the same in other linux distros. In ubuntu, the default username/password is trasnmission/transmission. I would have to do some tests with `$TR_AUTH` variable on other distros and see how it fits the best.

Since I do not own a mac hardware, would you be able to help with performing some tests on your mac, if required?

**Comment by mklbtz** [comment]
Yeah, I can certainly do that. I might need some help setting up the Python environment though. It's been a while. Also, please DM me on Twitter ([@mklbtz](https://twitter.com/mklbtz)) if you want me to respond quickly. I have to manually check these threads and I might forget 😅

**Comment by mschnetzer** [comment]
Hi, is there any news on this issue? Thanks!
